post-payments.php

```diff
@@ -149,6 +149,11 @@ public function get_report_data( $from_date, $to_date ) {
 					'value' => '0.00',
 					'compare' => '!=',
 				),
+				array(
+					'key' => $this->meta_key,
+					'value' => '',
+					'compare' => '!=',
+				),
```

Comment:
Every meta query you add will make it slower. You should refactor this to use `'compare' => 'NOT IN'` and make the value `array( '', '0.00' )`